src/layer/statisticspooling.cpp

```diff
@@ -0,0 +1,80 @@
+// Tencent is pleased to support the open source community by making ncnn available.
+//
+// Copyright (C) 2017 THL A29 Limited, a Tencent company. All rights reserved.
+//
+// Licensed under the BSD 3-Clause License (the "License"); you may not use this file except
+// in compliance with the License. You may obtain a copy of the License at
+//
+// https://opensource.org/licenses/BSD-3-Clause
+//
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+#include "statisticspooling.h"
+#include <float.h>
+#include <limits.h>
+#include <math.h>
+#include <algorithm>
+#include <functional>
```

Comment:
It seems these headers are not needed

```
#include <algorithm>
#include <functional>
```

Reply:
Thanks for your suggestions，I delete these headers.